addons/l10n_id_pos/models/qris_transaction.py

```diff
@@ -0,0 +1,18 @@
+from odoo import _, models
+from odoo.exceptions import ValidationError
+
+
+class QRISTransaction(models.Model):
+    _inherit = "l10n_id.qris.transaction"
+
+    def _constraint_model(self):
+        # add pos.order to list of possible models
+        if self.model not in ['account.move', 'pos.order']:
+            raise ValidationError(_("QRIS capability is not extended to model %s yet!", self.model))
```

Comment:
You should call super and only check `pos.order` since `account.move` is already checked in the super call